### ISSUE TITLE
Command "args:creates:" is ignored in async mode

### ISSUE THREAD

**Issue by hxw** [ansible/ansible#13828]
When command is used in async mode the args form of creates appears to be ignored.  However the embedded `creates=file` does work.

In the code below the first two items  (initialise,A) will _always_ report changed and their commands are run.  The third case (B) never runs.  Which shows that async code is only ignoring the args format. 

```
- name: testing initialise
  command: >-
    touch /tmp/testing_file
  args:
    creates: /tmp/testing_file
  async: 200
  poll: 10

- name: testing should not run A
  command: >-
    touch /tmp/testing_file
  args:
    creates: /tmp/testing_file
  async: 200
  poll: 10

- name: testing should not run B
  command: >-
    touch /tmp/testing_file
    creates=/tmp/testing_file
  async: 200
  poll: 10
```

**Comment by abadger** [comment]
This is fixed in 2.0:

```
$ ansible-playbook /srv/ansible/troubleshooting/13828.yml

PLAY ***************************************************************************

TASK [setup] *******************************************************************
ok: [localhost]

TASK [testing initialise] ******************************************************
changed: [localhost]
 [WARNING]: Consider using file module with state=touch rather than running touch


TASK [testing should not run A] ************************************************
ok: [localhost]

TASK [testing should not run B] ************************************************
ok: [localhost]

PLAY RECAP *********************************************************************
localhost                  : ok=4    changed=1    unreachable=0    failed=0
```

**Comment by bcoca** [comment]
closing as per above
